[Do not merge] Migrate APIs.guru related URLs

IvanGoncharov opened the pull request:
Hi @BigstickCarpet,

We are migrating our URLs to `APIs.guru` domain, here is more details:
https://github.com/APIs-guru/api-models/issues/85
Plus I add `Tested on APIs.guru` badge, hope you will like it :smile: 
[![Tested on APIs.guru](https://api.apis.guru/badges/tested_on.svg)](https://APIs.guru)

BigstickCarpet commented:
@IvanGoncharov - Thanks for the excellent PR, as always.  Apis.guru is looking good!  You've done a lot of excellent work there!   Also, I love the badge that you added.

Some of the Travis CI build are failing, and I think it has to do with the http-to-https redirect that occurs when visiting `http://api.apis.guru/v2/list.json`.  Can you try modifying the URL to be https (to avoid the redirect) and see if that fixes the builds?

IvanGoncharov commented:
@BigstickCarpet We removed `http` to `https` redirection.
But I think `https` is always better so I switched URL to it.

> Thanks for the excellent PR, as always. Apis.guru is looking good! You've done a lot of excellent work there! Also, I love the badge that you added.

Thank you, you was the first person who used APIs.guru.
And it was even before I implement API v1: https://github.com/BigstickCarpet/swagger-parser/pull/4#issuecomment-78648464

P.S. Still missing Twitter and in general have < 700 APIs. But I think we moving in right direction :)

IvanGoncharov commented:
@BigstickCarpet It looks like all problem is related to CloudFlare.
I can't disable their DDoS prevention which blocks PhantomJS.

I want to try S3 as a storage.
So I manually put API folder on S3 and will change API root in this PR.

BigstickCarpet commented:
Ok, cool.  I was wondering why the tests were failing, but haven't had time to look into it

BigstickCarpet commented:
Thanks again, @IvanGoncharov !!!

IvanGoncharov commented:
@BigstickCarpet `https://s3.amazonaws.com/api.apis.guru` just tests URL. I plan to use S3+CloudFront as the backend for `api.apis.guru`. But before that, I need to setup CI deploy + CloudFront + DNS.

So I can create new PR and continue testing or you can revert this PR.
Both scenarious works for me.

BigstickCarpet commented:
Oh.  Sorry.  I saw that all the tests finally passed on Travis CI, so I thought you were done with this PR.  That's why I merged it.

Can you create a new PR to continue your testing?  That'll be easier (for me :) ) than reverting this PR.

IvanGoncharov commented:
@BigstickCarpet Cool, I will create new PR. Your test suite is ideal for load testing since it's 5 tests in parallel doing async HTTP requests :)
